missions/serializers.py

```diff
@@ -1,6 +1,21 @@
 from rest_framework import serializers
 from .models import Chapter, Mission
 
+class MissionSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Mission
+        fields = [
+            "id",
+            "chapter",
+            "title",
+            "description",
+            "image",
+            "question_text",
+            "question_image",
+            "answer_assets",
+            "ai_prompt",
+        ]
```

Comment:
❗ 보안 측면에서 공격자가 `ai_prompt`를 알게 되면 악용할 수 있으므로, 이 데이터는 응답하지 않으면 좋겠습니다. `description`와 `image`는 학습단계 목록 페이지에서만 사용하는 정보로, 학습단계 상세 페이지에서는 사용하지 않으므로 응답하지 않아도 됩니다.

```suggestion
        fields = ("id","chapter","title","question_text","question_image","answer_assets",)
```

Reply:
오 생각하지 못한 점 리뷰 감사드립니다!!!